drivers/hmis_csv_importer/app/models/hmis_csv_importer/importer/importer.rb

```diff
@@ -829,6 +850,7 @@ def involved_project_ids
               # Client model doesn't have a uniqueness constraint because of the warehouse data source
               # so these must be processed more slowly
               if klass.hud_key == :PersonalID
+                Rails.logger.info "Processing #{batch.count} records individually for #{file_name}"
```

Comment:
Since you have `total_batches`, might as well update these to:
```suggestion
                Rails.logger.info "Processing #{batch.count} of #{total_batches} records individually for #{file_name}"
```

Reply:
total_batches calculation is removed in [this commit](https://github.com/greenriver/hmis-warehouse/pull/5975/commits/c7e47351177e19add346f2780bc1c71d26746299)